config/sync/migrate_plus.migration_group.default.yml

```diff
@@ -0,0 +1,10 @@
+uuid: da341a55-715f-4fa5-82e5-5f573b40de67
+langcode: en
+status: true
+dependencies: {  }
+id: default
+label: Default
+description: 'A container for any migrations not explicitly assigned to a group.'
```

Comment:
Where did this config come from?  It seems like this is for something else?

Reply:
It's just came out of D9 migrate plus upgrade. it kept showing up so i figured i'd commit it.